WorkloadManagementSystem/Client/PilotsLoggingClient.py

```diff
@@ -0,0 +1,43 @@
+""" Class that contains client access to the PilotsLogging handler. """
+
+__RCSID__ = "$Id$"
+
+from DIRAC.Core.Base.Client import Client
+
+class PilotsLoggingClient( Client ):
+  """Implementation of interface of Pilots Logging service. Client class should be used to communicate
+  with PilotsLogging Service
+  """
+
+  def __init__( self, **kwargs ):
+    Client.__init__( self, **kwargs )
+    self.setServer( 'WorkloadManagement/PilotsLogging' )
+    self.pilotsLoggingHandler = self._getRPC()
+
+  def addPilotsLogging( self, pilotUUID, timestamp, source, phase, status, messageContent ):
+    """
+    Add new Pilots Logging entry
+    :param pilotUUID: Pilot reference
```

Comment:
There must be an emptly line before the block with `:param...` otherwise they are not correctly rendered. This applies to all the other instances as well.